### DISCUSSION TITLE
Adding an extra field to the form > email config?

### DISCUSSION THREAD

**Discussion by dnrdigital** [statamic/cms#10418] [Q&A] [answered]
I have a website where content editors frequently create various forms for signing up to events, and sometimes need to add custom messages on a form-by-form basis to the confirmation emails that get sent to the form sender.

As the forms are created frequently, it's not practicable for me to create a new email template each time. Instead, I'd like give content editors a textarea field in the form config to add a custom message to the email body if they need to. 

Is there a straightforward way to do this? I've seen how to extend fieldtypes with appendConfigField but can't wrap my head around doing this for the email config field...

**Comment by jacksleight** [comment, marked as answer]
There's an open PR for this: https://github.com/statamic/cms/pull/8491

In the meantime I've used a global set to store those extra values before. Not as tidy as having it all in one place but it works.

**Comment by sjclark** [comment]
Personally I'd just create an additional form field with some fixed name like `email_blurb` - maybe put the text in the `instructions` part (can always use markdown), adjust the front-end template to not show that field, and then just adjust the email template to read that field into the text copy? Depends how much text you need really. But jacks suggestion would also work 🤷
